addons/nuxeo-multi-tenant/nuxeo-multi-tenant-core/src/test/java/org/nuxeo/ecm/multi/tenant/TestMultiTenantHelper.java

```diff
@@ -31,6 +31,7 @@
 @RunWith(FeaturesRunner.class)
 @Features(PlatformFeature.class)
 @RepositoryConfig(cleanup = Granularity.METHOD)
+@Deploy("org.nuxeo.ecm.platform.login")
```

Comment:
Why not NuxeoLoginFeature?

Reply:
Probably to avoid to deploy `web.common` during the development step. I will use the feature.